src/Interactive/EditorFeatures/Core/Extensibility/Interactive/InteractiveEvaluator.cs

```diff
@@ -248,9 +246,13 @@ private Dispatcher Dispatcher
 
         private static MetadataFileReferenceResolver CreateFileResolver(ImmutableArray<string> referencePaths, string baseDirectory)
         {
+            var userProfilePath = Environment.GetEnvironmentVariable("userprofile");
```

Comment:
Enviroment.GetFolderPath(Environment.SpecialFolder.UserProfile) instead? I don't think that's guaranteed to exist.